### ISSUE TITLE
Form::group (add method??) messing up the next row style(s)

### ISSUE THREAD

**Issue by edwinheij** [inkvizytor/FluentForm#6]
``` php
{!! Form::group()
    ->add(
        Form::text('birthdate', Input::get('birthdate'))
            ->placeholder('Geboortedatum')
            ->prepend(Fluent::icon(Fluent::FA_CALENDAR))
            ->css('datepicker')
    )->label('Geboortedatum')
!!}
```

WIll damage the next row styles? (div too much?)

![screenshot from 2015-11-02 22-22-25](https://cloud.githubusercontent.com/assets/1799101/10894839/0bb63030-81b1-11e5-932e-2216c00d72d9.png)

**Comment by edwinheij** [comment]
when removing 'prepend' styles are OK.
--edit--
when using 'prepend' WITH Fluent::icon(...) the next form::group() (multiple in my case)

``` php
{!! Form::group()->text('street', Input::get('street'))->label('Street')->placeholder('Street') !!}
```

won't have `<div class="form-group">` anymore.
When using text in the prepend method it all goes well.

**Comment by inkvizytor** [comment]
I think i have fixed the problem, try version 0.8.7.

**Comment by edwinheij** [comment]
nice work! Fixed. Thanks
